gemini/use-cases/retrieval-augmented-generation/utils/intro_multimodal_rag_utils.py

```diff
@@ -387,7 +364,16 @@ def get_image_for_gemini(
 
 
 def get_gemini_response(
-    generative_multimodal_model, model_input: List[str], stream: bool = False
+    generative_multimodal_model,
+    model_input: List[str],
+    stream: bool = True,
+    generation_config: Optional[dict] = {"max_output_tokens": 2048, "temperature": 0.2},
```

Comment:
Can you change this to use the `GenerationConfig` class?

https://cloud.google.com/python/docs/reference/aiplatform/latest/vertexai.preview.generative_models.GenerationConfig

Reply:
Changed!